libs/wingsdk/src/shared-aws/function.inflight.ts

```diff
@@ -27,7 +27,10 @@ export class FunctionClient implements IFunctionClient {
     if (!response.Payload) {
       return "";
     }
-    const value = JSON.parse(toUtf8(response.Payload));
+    const value =
+      toUtf8(response.Payload) === "null"
+        ? ""
+        : JSON.parse(toUtf8(response.Payload));
```

Comment:
This will result in the result being be parsed into utf8 twice.

If the string is "null" then `JSON.parse("null")` will equal `null` in JS, so I think this might also work:

```suggestion
    const value = JSON.parse(toUtf8(response.Payload)) ?? "";
```